py3status/core.py

```diff
@@ -419,6 +424,19 @@ def notify_update(self, update):
             if group_module:
                 group_module['module'].force_update()
 
+    def log(self, msg, level='info'):
+        """
+        log this information to syslog or user provided logfile.
+        """
+        if not self.config['log_file']:
+            # If level was given as a str then convert to actual level
+            level = LOG_LEVELS.get(level, level)
+            syslog(level, msg)
+        else:
+            with open(self.config['log_file'], 'a') as f:
+                log_time = time.strftime("%Y-%m-%d %H:%M:%S")
+                f.write('{}\t{}\t{}\n'.format(log_time, level.upper(), msg))
```

Comment:
tabs on log files ? I'd strongly suggest avoiding that please, simple space is enough